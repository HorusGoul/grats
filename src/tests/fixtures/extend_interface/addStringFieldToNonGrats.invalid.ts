type SomeNonGratsType = string;

/** @gqlField */
export function greeting(someType: SomeNonGratsType): string {
  return `Hello ${someType}!`;
}
