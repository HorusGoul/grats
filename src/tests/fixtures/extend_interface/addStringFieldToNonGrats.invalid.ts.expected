-----------------
INPUT
----------------- 
type SomeNonGratsType = string;

/** @gqlField */
export function greeting(someType: SomeNonGratsType): string {
  return `Hello ${someType}!`;
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/extend_interface/addStringFieldToNonGrats.invalid.ts:4:36 - error: Unable to resolve type reference. In order to generate a GraphQL schema, Grats needs to be able to resolve all type references to their declaration. However this reference could not be resolved. Is it possible that this type is not defined in this file?

4 export function greeting(someType: SomeNonGratsType): string {
                                     ~~~~~~~~~~~~~~~~
